### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by jborgers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ PMD-jPinpoint-rules is licensed under the [Apache License, Version 2.0](https://
 To use the ruleset you can install: 
 
 - the PMD command line tool from [PMD project at github](https://pmd.github.io/) and/or
-- the PMDPlugin in you development environment. 
+- the PMD-Plugin in you development environment.
```
Comment (stokpop) [inline README.md:41]: typo: in your

File: README.md [hunk 2]
```diff
@@ -58,19 +58,20 @@ After installing the tool you can run `pmd.sh` or `pmd.bat` similar to the follo
 
 - Next, configure (add) the ruleset from this repo by URL to always be up-to-date:
 
-      Settings > Other Settings > PMD > RuleSets > + > Choose RuleSet > drop down > Choose 'jpinpoint-rules' 
+      Settings > Tools > PMD > RuleSets > + > Choose RuleSet > drop down > Choose 'jpinpoint-rules' 
 
 - Alternatively, download and add your local copy: *rulesets/java/jpinpoint-rules.xml*
```
Comment (stokpop) [inline README.md:63]: can turn this into a link? As here:

[rulesets/java/jpinpoint-rules.xml](https://raw.githubusercontent.com/jborgers/PMD-jPinpoint-rules/refs/heads/pmd7/rulesets/java/jpinpoint-rules.xml)

```text
[rulesets/java/jpinpoint-rules.xml](https://raw.githubusercontent.com/jborgers/PMD-jPinpoint-rules/refs/heads/pmd7/rulesets/java/jpinpoint-rules.xml)
```

File: README.md [hunk 3]
```diff
@@ -58,19 +58,20 @@ After installing the tool you can run `pmd.sh` or `pmd.bat` similar to the follo
...
+  - remember to download regularly to get up-to-date
   
-- Options tab: check 'Skip Test Sources' and set your Java version 
+- Options tab: optionally check 'Skip Test Sources' and optionally set your Java version 
 
 - You can now perform the code checks using [right-click] on a folder or a file and choose:
```
Comment (stokpop) [inline README.md:68]: we can add some screenshots (later?)

File: README.md (hunk 2 above)
Comment (stokpop) [reply README.md:63]: Add this link also at Eclipse section

**Review by stokpop — CHANGES_REQUESTED** [review]
I also see some docs about Kotlin and PMD-7.0.0-SNAPSHOT branch, I think that also needs updates.

File: README.md (hunk 3 above)
Comment (jborgers) [reply README.md:68]: good idea

Comment (jborgers) [reply README.md:68]: good idea

**Pull request merged**